mephisto/client/review_server.py

```diff
@@ -14,22 +14,20 @@
 import threading
 
 
-if False:  # sys.stdout.isatty():
-    pass
-else:
-    # disable all logging out to stdout
-    import logging
-
-    log = logging.getLogger("werkzeug")
-    log.setLevel(logging.ERROR)
-    cli = sys.modules["flask.cli"]
-    cli.show_server_banner = lambda *x: None
-
-
-def run(build_dir, port):
+def run(build_dir, port, output, csv_headers, debug=False):
     global index_file, app
     global ready_for_next, current_data, finished, index_file
     global counter
+
+    if not debug or output == "":
+        # disable noisy logging of flask
+        import logging
+
+        log = logging.getLogger("werkzeug")
+        log.disabled = True
```

Comment:
nit: naming can make this code more self-documenting. `flask_log` makes the implicit knowledge that the flask logger's name is `werkzueg` than just calling it `log`.

Reply:
Good suggestion! Much more clear than this stack overflow copy pasta